localgov_alert_banner.info.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
   - drupal:field
   - drupal:field_group
   - drupal:link
+  - drupal:node
```

Comment:
Not important, I think it's pretty rare not to have node, but does this depend on it?

Reply:
Required by this in AlertBannerForm.php
````php
// Create the vertical tabs like on node edit forms.
// @src https://drupal.stackexchange.com/a/276907
$form['#theme'][] = 'node_edit_form';
$form['#attached']['library'] = ['node/drupal.node'];
````